pyproject.toml

```diff
@@ -71,7 +71,14 @@ packages = ["src/snowcli", "src/templates"]
 line-length = 88
 select = [
     "N",
-    "I"  # isort
+    "I",  # isort
+    "G",  # flake8-logging-format
```

Comment:
Reading up on this, [the spec](https://pypi.org/project/flake8-logging-format/) says that part of the motivation is to make sure we can redact logs. Do we have the other pieces in place to make this happen? Is that our motivation for adopting this too? I am a big f-string fan, but if they're not doing what we need them to do, I agree with the change. Unfortunately without this context the change to using e.g. `%s` feels like a step backwards in code readability.

Reply:
This applies only to logging. Using `%s` is a common practices in python as rendering the string is defferred. Here are docs: https://docs.python.org/3/howto/logging.html#optimization

Reply:
Sounds good -- so for us the main argument is optimization. I suppose getting rid of f-strings even when not necessary (e.g. for things that aren't heavy to compute) is just an unfortunate side-effect of the linter's abilities.

Reply:
Apart from optimisation you can create a custom formatter that for example highlights "input" to log message. This wouldn't be possible with f-strings